GitUI/Editor/FileViewer.cs

```diff
@@ -73,6 +73,7 @@ private enum ViewMode
         private Func<Task>? _deferShowFunc;
         private readonly ContinuousScrollEventManager _continuousScrollEventManager;
         private FileStatusItem? _viewItem;
+        private bool _confirmResetLines = true;
```

Comment:
This should be marked as test specific, a proposal below
Even better would be to hide this in MessageBox.Show() instead with a test accessor
(assuming that there is no short term way to avoid this)
```suggestion

        // This variable is used by tests to avoid popups
        private bool _unitTestConfirmResetLines = true;
```

Reply:
`MessageBox.Show()` is not under our control and making a wrapper for it seems a daunting task for me at least :)
I like your suggested rename, thanks!

Reply:
Wrap the call to MessageBox.Show() instead or rather switch to TaskDialog

Reply:
Have a look at `Use TaskDialog instead of MessageBox.Show for reset selected lines confirmation`. Thoughts?